docpages/example_programs/the_basics.md

```diff
@@ -7,3 +7,4 @@ These example programs are great to get started with simple things in the D++ li
 * \subpage attach-file "Attaching a file"
 * \subpage webhooks "Webhooks"
 * \subpage callback-functions "Using Callback Functions"
+* \subpage editing-channel-end-message
```

Comment:
`editing-channel-end-message`? Change to `editing-channel-and-message` :)

Reply:
Uh... I mean, I am impressed with myself.